packages/protocol/contracts/governance/Validators.sol

```diff
@@ -483,6 +520,7 @@ contract Validators is IValidators, Ownable, ReentrancyGuard, Initializable, Usi
     require(voters[account] == address(0));
     uint256 weight = getAccountWeight(account);
     require(weight > 0);
+    totalVotes = totalVotes.add(weight);
```

Comment:
Not quite this simple - `vote` also allows you to change your vote, which wouldn't increase `totalVotes`. I think moving to `else` clause below should do the trick.

Reply:
I think you have to call `revokeVote` to change the vote, otherwise `require(voters[account] == address(0))` ?

Reply:
Ah, you're right, the condition below is about whether or not this group has been voted for before.